agent/src/agent/scripts/send_watermarks.py

```diff
@@ -18,6 +18,12 @@ def _update_errors_count(num_of_errors):
     return num_of_errors + 1
 
 
+def _if_send_watermark(pipeline_: pipeline.Pipeline):
```

Comment:
I'd rather call it _should_send_watermark, because now it's like `if _if_send..()`, tautology